source/draw/gpu/opengl/AndroidContext.ooc

```diff
@@ -75,15 +75,14 @@ AndroidContext: class extends OpenGLContext {
 		fence := this createFence()
 		fence sync()
 		eglImage := gpuRgba as EGLBgra
-		sourcePointer := eglImage buffer lock(false)
+		sourcePointer := eglImage buffer lock(false) as UInt8*
```

Comment:
Doesn't need to be there, I added it now for clarity. Shouldn't really be part of this PR so I can remove it if it's important.